docs/topics/mta-web-config-maven-rep.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * docs/rules-development-guide/master.adoc
+
+:_content-type: PROCEDURE
+[id="mta-web-config-maven-rep_{context}"]
+= Configuring Maven repositories
+
+This view provides general options for the Maven repositories, such as managing the size of a local repository or configuring how to interact with external repositories.
+
+== Managing repository size
+
+.Procedure
+
+. In the *Administrator* view, click *Repositories* and then *Maven*.
```

Comment:
Better to repeat "click" -- that is.

Reply:
In the Administrator view, click Repositories and then click Maven.